package no.nav.syfo.shared.util.exception

class ConflictException(message: String) : RuntimeException(message)
